ui/components/LoginDialog.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div>
+    <v-dialog
+      width="500"
+      :value="visible"
+      @input="$emit('update:visible', false)"
+    >
+      <v-card>
+        <v-card-title class="headline grey lighten-2" primary-title>
+          Login
+        </v-card-title>
+
+        <v-card-text class="pt-4">
+          <div v-if="errorMessage" class="error-message">
+            {{ errorMessage }}
+          </div>
+          <v-form>
+            <v-text-field id="login-input" v-model="form.login" label="Login" />
+            <v-text-field
+              id="password-input"
+              v-model="form.password"
+              label="Password"
+            />
+          </v-form>
+        </v-card-text>
+
+        <v-divider />
+
+        <v-card-actions>
+          <v-spacer></v-spacer>
+          <v-btn color="primary" text @click="login">
```

Comment:
@lusarz Because you use just a `btn`, without input type set to `submit`, you'll not be able to submit the form using keyboard.